### ISSUE TITLE
mrfioc2 software version and git tag

### ISSUE THREAD

**Issue by Insomnia1437** [epics-modules/mrfioc2#145]
When I read `$(P)SwVer-I` PV, I found the value becomes something like: `kmod-deb-3-0-2-gd1f0ec7f-dirty`  rather than `2.6.0`

The MRF_VERSION  macro is generated from  `genVersionHeader.pl`
```
perl -CSD ../genVersionHeader.pl -t "../../.." -V "2.6.0" -N MRF_VERSION version.h
Creating VCS version header version.h with "kmod-deb-3-0-13-gd1f0ec7f-dirty"
```
inside `genVersionHeader.pl`, it uses the most recent tag and current commit ID to denote the mrfioc2 version

https://github.com/epics-modules/mrfioc2/blob/5b2109655ea801c904e507abcae02d8881a585db/mrfCommon/src/genVersionHeader.pl#L64-L72

Perhaps we should remove these two tags or modify `genVersionHeader.pl` to keep digit tags only
- [kmod-deb-3-0](https://github.com/epics-modules/mrfioc2/releases/tag/kmod-deb-3-0)
- [kmod-rpm-3-1](https://github.com/epics-modules/mrfioc2/releases/tag/kmod-rpm-3-1)

**Comment by jerzyjamroz** [comment]
@Insomnia1437 , ```kmod``` releases are not the IOC releases. There is the IOC artefact so the ci is for the upgrade in the future.

**Comment by Insomnia1437** [comment]
fix in #146 
close this issue
